snuba/migrations/snuba_migrations/transactions/0015_transactions_add_source_column.py

```diff
@@ -0,0 +1,50 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import Column, String
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    """
+    Add source column required for ...
```

Comment:
The source column is required to query for transaction events in the transaction summary when the transaction name has been redacted in metrics.